src/main/java/net/atos/zac/app/planitems/PlanItemsRESTService.java

```diff
@@ -141,7 +146,8 @@ public void doUserEventListenerPlanItem(final RESTUserEventListenerData userEven
                 final PlanItemInstance planItemInstance = caseService.readOpenPlanItem(userEventListenerData.planItemInstanceId);
                 caseVariablesService.setOntvankelijk(planItemInstance.getCaseInstanceId(), userEventListenerData.zaakOntvankelijk);
                 if (!userEventListenerData.zaakOntvankelijk) {
-                    final ZaakafhandelParameters zaakafhandelParameters = zaakafhandelParameterService.readZaakafhandelParameters(zaak);
+                    final ZaakafhandelParameters zaakafhandelParameters =
+                            zaakafhandelParameterService.readZaakafhandelParameters(UriUtil.uuidFromURI(zaak.getZaaktype()));
```

Comment:
Zaaktype heeft een convenience method: getUUID() Dan is die UriUtil.uuidFromURI(zaak.getZaaktype()) niet nodig.

Reply:
is een Uri ipv een Zaaktype